eng/targets/Helix.Common.props

```diff
@@ -28,7 +29,7 @@
       <!-- aspnetcore-ci[-official] (ci.yml) -->
       <!-- aspnetcore-quarantined-pr (quarantined-pr.yml) -->
       <ItemGroup>
-        <HelixAvailableTargetQueue Include="Ubuntu.2004.Amd64.Open" Platform="Linux" />
+        <HelixAvailableTargetQueue Include="Ubuntu.2204.Amd64.Open" Platform="Linux" />
```

Comment:
I see you reverted the Ubuntu2004 -> 2204 change in Helix.targets - did you mean to revert this one as well?

Reply:
Yes.  That one will require some investigation and change. I was wanting to see if we could make a clean PR w/o that change.